packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart

```diff
@@ -156,12 +161,19 @@ class DevServerRunner {
     logger.detail('[codegen] complete.');
   }
 
-  Future<void> _reload() async {
-    logger.detail('[codegen] reloading...');
+  Future<void> _reload([bool verbose = false]) async {
+    final void Function(String) log;
+    if (verbose) {
+      log = logger.info;
+    } else {
+      log = logger.detail;
```

Comment:
**Nit & Question**: Why are we preferring to use `logger.info` or `logger.detail` over `logger.progress` to indicate the progress of the reload?

Reply:
That's an idea for a further PR

Reply:
Okay, sounds good to me. I thought it was intentional.